gitops/TravisCi/README.md

```diff
@@ -0,0 +1,151 @@
+# Approach
+
+Taking into account that devops-as-code provides convenient wrapper for *NIX and Windows systems, it is very easy to using together with different CI tools.
+
+To integrate devops-as-code into your git flow, please follow these simple steps:
+
+1) Add wrappers to your project. You can either copy/paste them from another project, or execute `xl wrapper` command in the root of your project to generate them.
+2) Define devops-as-code YAML in your project's directory.  
+3) Define `sh / bat` step in your `.travis.yaml` and point to your devops-as-code YAML file(s)
```

Comment:
It's better to change `sh / bat` to `bat / sh` to be consistent with the example.